pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -979,6 +979,11 @@ type NodeGroup struct {
 	// ContainerRuntime defines the runtime (CRI) to use for containers on the node
 	// +optional
 	ContainerRuntime *string `json:"containerRuntime,omitempty"`
+
+	// MaxInstanceLifetime defines the maximum amount of time in seconds an instance stays alive.
+	// To clear a previously set value, this must be set to 0 explicitly.
+	// +optional
+	MaxInstanceLifetime *int `json:"maxInstanceLifetime,omitempty"`
```

Comment:
is providing this in seconds practical :thinking: I know thats what they want in the stack, we can always convert a more meaningful value into seconds. Perhaps use the `time` type? Although looking behind scenes thats only ` Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`, but providing it this way might be slightly easier?

Reply:
I disagree. I would like to be consistent with what CF provides, because that's well documented. If you change that that's unexpected, which means, you have to explicitly document it so it's apparent which means more burden on the user to discover stuff for the minor addition of using hours instead of secods.

Reply:
> I disagree. I would like to be consistent with what CF provides

folks who use eksctl are necessarily all that familiar with CF 

> because that's well documented. If you change that that's unexpected, which means, you have to explicitly document it so it's apparent which means more burden on the user to discover stuff for the minor addition of using hours instead of secods.

they would have to discover this field in our docs regardless?

I'm open to leaving it as is, but if a day is the minimum value do we expect people to really want the granular control of seconds? Or is setting it in multiples of days/hours a better UX